### PR TITLE
Add thumbnail images to the external resources page

### LINKED ISSUE #1936: Add thumbnail images to the external resources page
What do people think about adding thumbnail images to this external resources page? I'm thinking of using 'cards' like in the gallery/team/overview page, and having a thumbnail of the linked repository or youtube page so people know what is in those links before they click. Can be done in a separate PR.

_Originally posted by @weiji14 in https://github.com/GenericMappingTools/pygmt/pull/1935#pullrequestreview-988530738_

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

TODO:

- [x] Update thumbnail images

Fixes #1936.

### CODE REVIEW

File: doc/external_resources.md [hunk 1]
```diff
@@ -10,14 +10,91 @@ to submit a pull request with your recommended addition to the
...
-- [GMT and PyGMT plotting examples](https://github.com/michaelgrund/GMT-plotting) by Michael Grund
-- [NZ Antarctic Science Conference 2021 poster](https://github.com/weiji14/nzasc2021) by Wei Ji Leong
+:::::{grid} 1 2 2 3
+
+::::{grid-item-card} GMT and PyGMT plotting examples
+:link: https://github.com/michaelgrund/GMT-plotting
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
```
Comment (seisman) [inline doc/external_resources.md:85]:
```suggestion
![](https://user-images.githubusercontent.com/23025878/59599891-f439ce00-90ff-11e9-82be-5e324fbcc893.png)
```

File: doc/external_resources.md [hunk 2]
```diff
@@ -10,14 +10,91 @@ to submit a pull request with your recommended addition to the
...
++++
+Wei Ji Leong, Leonardo Uieda, Max Jones, Andre Belem
+::::
+
+::::{grid-item-card} 2021 PyGMT course at the UAF Geophysical Institute
+:link: https://github.com/liamtoney/gi-pygmt-2021
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
```
Comment (seisman) [inline doc/external_resources.md:30]:
```suggestion
![](https://user-images.githubusercontent.com/3974108/172042115-3f700c16-0f78-49d3-9e8d-87c77e4c0444.png)
```

File: doc/external_resources.md [hunk 3]
```diff
@@ -10,14 +10,91 @@ to submit a pull request with your recommended addition to the
...
++++
+MIGG-NTU
+::::
+
+::::{grid-item-card} PyGMT Workshop at FOSS4G Oceania 2019
+:link: https://github.com/GenericMappingTools/foss4g2019oceania
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
```
Comment (weiji14) [inline doc/external_resources.md:70]: Use this thumbnail instead
```suggestion
![](https://github.com/GenericMappingTools/foss4g2019oceania/releases/download/v1/picton_3d_dsm_view.png)
```

File: doc/external_resources.md [hunk 4]
```diff
@@ -10,14 +10,91 @@ to submit a pull request with your recommended addition to the
...
++++
+Liam Toney
+::::
+
+::::{grid-item-card} PyGMT Tutorial in 2021
+:link: https://github.com/MIGG-NTU/PyGMT2021
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://user-images.githubusercontent.com/3974108/171653272-efaceeb1-a253-44a1-bddc-36bc9498c19b.png)
```
Comment (weiji14) [inline doc/external_resources.md:60]: Need a proper thumbnail, let me try and run the notebook using https://mybinder.org/v2/gh/MIGG-NTU/PyGMT2021/master and see if I can get one up.

Comment (weiji14) [reply doc/external_resources.md:60]: Hmm, the figures seem to be about identical to the PyGMT gallery examples. Perhaps we just add a screenshot of the GitHub repo at https://github.com/MIGG-NTU/PyGMT2021?

```suggestion
![](https://user-images.githubusercontent.com/23487320/172058757-9b68f7ae-7a3f-4f48-9df6-9a065d6cd448.png)
```

image

**Review by weiji14 — APPROVED** [review]
Awesome work! Much nicer to see what to click on!

image

**Pull request merged**